src/analytics/parameter.ts

```diff
@@ -451,7 +451,7 @@ export function createStackParameters(scope: Construct): {
             'ClusterIdentifier and DbUser are required when using Redshift Provisioned cluster.',
       },
     ],
-  });
+  }).overrideLogicalId('RedshiftProvisionedParameters');
```

Comment:
to avoid the sonarqube warning.
<img width="997" alt="image" src="https://github.com/awslabs/clickstream-analytics-on-aws/assets/79438022/ce650f3a-b9ee-4297-a48d-f56392d783d2">